_episodes/02-introduction-to-agile.md

```diff
@@ -40,10 +46,10 @@ Learn more about the waterfall model [here](https://www.tutorialspoint.com/sdlc/
 
 # Agile manifesto
 The Agile way of working contrasts the Waterfall model and mitigates most of its drawbacks.
-Watch [this video introducing the agile manifesto](http://growingagile.thinkific.com/courses/take/scrumbasics/lessons/967803-agile-manifesto-lesson).
+Watch [this video introducing the Agile manifesto](http://growingagile.thinkific.com/courses/take/scrumbasics/lessons/967803-agile-manifesto-lesson).
```

Comment:
@SarahAlidoost Is it normal that this link to the video takes you to a page where you need an account to log in?

Reply:
Yes this is normal. We mention making an account in the setup instructions.

Reply:
Ah yes I hadn't seen that section